clients/githubrepo/search.go

```diff
@@ -59,7 +59,7 @@ func (handler *searchHandler) buildQuery(request clients.SearchRequest) (string,
 	}
 	var queryBuilder strings.Builder
 	if _, err := queryBuilder.WriteString(
-		fmt.Sprintf("%s repo:%s/%s", request.Query, handler.owner, handler.repo)); err != nil {
+		fmt.Sprintf("%s repo:%s/%s", strings.ReplaceAll(request.Query, "/", " "), handler.owner, handler.repo)); err != nil {
```

Comment:
Nit: Provide the GitHub link in comment - https://docs.github.com/en/search-github/searching-on-github/searching-code#considerations-for-code-search for reference.

Reply:
This *may* not fully solve the problem. We should consider maintaining a list of special characters from https://docs.github.com/en/search-github/searching-on-github/searching-code#considerations-for-code-search to replace with space.

Reply:
It solves the problem for github URL since they can only have `-` and alphanums, but I suppose it's possible scorecard will later to more complex queries.